exporters/otlp/internal/transform/attribute.go

```diff
@@ -74,10 +74,170 @@ func toAttribute(v kv.KeyValue) *commonpb.KeyValue {
 		result.Value.Value = &commonpb.AnyValue_StringValue{
 			StringValue: v.Value.AsString(),
 		}
+	case kv.ARRAY:
+		result.Value.Value = toArrayAttribute(v)
 	default:
 		result.Value.Value = &commonpb.AnyValue_StringValue{
 			StringValue: "INVALID",
 		}
 	}
 	return result
 }
+
+func toArrayAttribute(v kv.KeyValue) *commonpb.AnyValue_ArrayValue {
+	array := v.Value.AsArray()
+	resultValues := []*commonpb.AnyValue{
+		{
+			Value: &commonpb.AnyValue_StringValue{
+				StringValue: "INVALID",
+			},
+		},
+	}
+	if boolArray, ok := array.([]bool); ok {
```

Comment:
Type assertions are cheaper then reflection, and also safer as we don't have to compare type names.
However, it would still be nice to avoid it somehow. I couldn't think of any abstraction, as the types defined in the commons proto are specifically distinct (even the field names are different). does anyone have any ideas?